lib/index.js

```diff
@@ -23,7 +27,16 @@ log.info('Get these values from your Rave Dashboard. For optional fields, just c
 prompt.start();
 prompt.getAsync(Object.keys(schema))
   .then((result) => Joi.validateAsync(result, schema))
-  .then(value => fs.writeFileAsync(configFile, JSON.stringify(value, null, 2) + '\n'))
+  .then((value) => {
+    if (value.liveMode === 'y') {
+      raveUri = raveLiveUri;
+    }
+    if (value.liveMode === 'n') {
```

Comment:
How about defaulting to test mode here. So that... if it's not live mode, the test mode is used as default. 

So.. we can have at the top (line 19) `let raveUri = raveTestUri;` Then, we'll only check for `y` and override to `raveLiveUri`...

We can then update the Readme to reflect test mode as default.